task.sql

```diff
@@ -38,3 +38,16 @@ CREATE TABLE ProductInventory (
     PRIMARY KEY (ID)
 );
 --rollback DROP TABLE ProductInventory;
+
+--changeset fredy.zoba:1 labels:0.0.2
+CREATE TABLE Users (
+    ID INT PRIMARY KEY AUTO_INCREMENT,
```

Comment:
The `AUTO_INCREMENT` attribute is correctly used here with the `PRIMARY KEY`. Ensure that the database engine supports `AUTO_INCREMENT` (e.g., MySQL).